app-examples/car-management-hexagonal-arch/domain/src/commonMain/kotlin/dev/eskt/example/domain/process/CarDriverLogger.kt

```diff
@@ -11,8 +11,9 @@ import dev.eskt.store.test.w.driver.DriverRegisteredEvent
 import dev.eskt.store.test.w.driver.DriverStreamType
 
 @EventListener
-class CarDriverLogger: MultiStreamTypeEventListener<Any, Uuid> {
+class CarDriverLogger : MultiStreamTypeEventListener<Any, Uuid> {
     override val id: String = "car-driver-pm"
+    val logs = mutableListOf<String>()
```

Comment:
this should probably be a port in the domain, and then you can inject an in-memory list for tests and something else in the actual prod code.

Reply:
What? I don't understand at all. maybe this is badly named. But I just wanted some way to assert the test results

Reply:
code in `app-examples/car-management-hexagonal-arch/domain` should try to be a reasonable real-life domain, and you would never have state inside a class called `CarDriverLogger`.
I'm proposing you introduce some sort of `Logger` domain port (interface) and in your test you create a `ListLogger` that allows you to verify the data, while the prod code implements an `StdOutLogger`.

We do something similar to the other PM in the same package.

Reply:
I see! Updated